markovflow/sde/linearize_sde.py

```diff
@@ -0,0 +1,200 @@
+#
+# Copyright (c) 2021 The Markovflow Contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Module to linearize a non-linear SDE."""
+
+import tensorflow as tf
+from gpflow.quadrature import mvnquad
+from gpflow.base import TensorType
+
+from markovflow.base import APPROX_INF
+from markovflow.sde import SDE
+
+
+class LinearizeSDE(SDE):
```

Comment:
The functionality looks right to me.

I m not sure having a LinearizeSDE class that inherits from the SDE makes sense.

We have a SSM class that already could be wrapper for As,bs,Qs